src/components/MessageInput/index.ts

```diff
@@ -0,0 +1,28 @@
+import Block from '../../utils/Block';
+import template from './messageInput.pug';
+import './messageInput.scss';
+
+export interface MessageInputProps {
+  label: string;
+  type: string;
```

Comment:
обычно тип чего-то - это ограниченный набор значений, поэтому, чтобы не передать ему любую строку, можно задать его тип в виде множества
```suggestion
  type: 'type1' | 'type2 | и тд';
```